raiden/tests/benchmark/conftest.py

```diff
@@ -0,0 +1,90 @@
+import contextlib
+import shlex
+import subprocess
+import time
+
+import pytest
+
+from raiden.tests.integration.fixtures.blockchain import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.raiden_network import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.smartcontracts import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.transport import *  # noqa: F401,F403
+
+from . import _codespeed
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--codespeed-url",
+        action="store",
+        default=None,
+        help="the URL of a running Codespeed instance, " "e.g. http://127.0.0.1:8000",
+    )
+    parser.addoption(
+        "--output-benchmark-results",
+        action="store",
+        default=None,
+        help="the path to write benchmark results to",
+    )
+
+
+def pytest_collection_modifyitems(items):
+    for item in items:
+        item.add_marker(pytest.mark.asyncio)
+
+
+_TIMES = {}
+
+
+_COMMIT_ID = (
+    subprocess.Popen(
+        shlex.split("git describe --tags --abbrev=8"),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+    .communicate()[0]
```

Comment:
`subprocess.check_output` would be more concise and raise on errors. Any reason for not using it?

Reply:
Just didn't think of it. Thanks for the suggestion, changed.